### PR TITLE
BLEN-57: [Community] Error with Attic-Demo-CCBY scene.

### PR DESCRIPTION
**Pull request by DagerD**
### PURPOSE
Fix exception with Round shader node.

### EFFECT OF CHANGE
Render works properly with Round shader node.

### TECHNICAL STEPS
Fixed wrong using of "if_else" method.

### CODE REVIEW

File: src/hdusd/bl_nodes/nodes/converter.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def export(self):
             res = in1.ceil()
         elif op == 'ROUND':
             f = in1.floor()
-            res = (in1 % 1.0 < 0.5).if_else(f, f + 1.0)
+            res = (in1 % 1.0).if_else('>=', 0.5, f + 1.0, f)
```
Comment (bsavery) [inline src/hdusd/bl_nodes/nodes/converter.py:50]: 0.5 rounds down to 0, 0.51 rounds up to 1. 

I think you want > not >=

Comment (DagerD) [reply src/hdusd/bl_nodes/nodes/converter.py:50]: No, with in1=0.5 it rounds to 1.
image

**Review by bnagirniak — APPROVED** [review]
Looks correct

**Review by VascoPi — COMMENTED** [review]
Discussed offline.

**Review by VascoPi — APPROVED** [review]
Works well.

**Pull request merged**